CHANGELOG/CHANGELOG-1.7.md

```diff
@@ -1,5 +1,18 @@
 # Changelog 1.7
 
+## 1.7.3 - 2023-03-28
+
+### Added
+
+- Set NO_FLOOD to IPsec tunnel ports. ([#4419](https://github.com/antrea-io/antrea/pull/4419), [@xliuxu])
+
+### Fixed
+
+- Fix that Service routes may get lost when starting on Windows. ([#4470](https://github.com/antrea-io/antrea/pull/4470), [@hongliangl])
+- Restore NO_FLOOD to OVS ports after reconnecting the OVS bridge. ([#4654](https://github.com/antrea-io/antrea/pull/4654), [@xliuxu])
+- Ensure OVS port is valid before setting NO_FLOOD. ([#4674](https://github.com/antrea-io/antrea/pull/4674), [@tnqn])
+- Fix route deletion for Service ClusterIP and LoadBalancerIP. ([#4711](https://github.com/antrea-io/antrea/pull/4711), [@tnqn])
```

Comment:
You can use the descriptions for these PRs in newer releases to be consistent and to avoid another round of review of the same items.

For example: https://github.com/antrea-io/antrea/blob/6590c2adeef85c6b234ec5775caa7a612b16250f/CHANGELOG/CHANGELOG-1.11.md?plain=1#L44

And 4419, 4654, 4674 should be merged to one item like it's done in v1.11 as they are for the same purpose.

Reply:
Done